src/hackerspace_online/tests/test_views.py

```diff
@@ -83,3 +83,9 @@ def test_favicon_public_tenant(self, mock_connection1, mock_connection2):
 
     def test_password_reset_view(self):
         self.assert200('account_reset_password')
+    
+    def test_achievements_redirect_to_badges_view(self):
+        user = User.objects.create_user(username="test_user", password="password")
+        self.client.force_login(user)
+        response = self.client.get('/achievements/')
+        self.assertRedirects(response, reverse("badges:list"))
```

Comment:
Can you add another test that going to "/achievements/1" (or the detail view for some other badge that exists) still works (either works directly ior redirects to "/badges/1"?  It's not clear to me from the redirect that it will work, might only redirect to list view, which mean broken links to any badge detail view... possibly.  Test would add confidence it work.